engine/lite/kafka/runtime/src/test/scala/pl/touk/nussknacker/engine/lite/kafka/KafkaTransactionalScenarioInterpreterTest.scala

```diff
@@ -46,6 +46,28 @@ class KafkaTransactionalScenarioInterpreterTest extends fixture.FunSuite with Ka
 
   private def withMinTolerance(duration: Duration) = duration.toMillis +- 60000L
 
+  test("should have same timestamp on source and sink") { fixture =>
+
+    val inputTopic = fixture.inputTopic
+    val outputTopic = fixture.outputTopic
+    val scenario = passThroughScenario(fixture)
+    val inputTimestamp = System.currentTimeMillis()
+
+    runScenarioWithoutErrors(fixture, scenario) {
+      val input = "test-input"
+      kafkaClient.sendRawMessage(
+        inputTopic,
+        key = null,
+        content = input.getBytes(),
+        timestamp = inputTimestamp
+      ).futureValue
```

Comment:
why do you invoke futureValue ?

Reply:
removed it, it was just in case but turned out unnecessary